rewrite-gradle/src/test/java/org/openrewrite/gradle/EnableDevelocityBuildCacheTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.gradle;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.gradle.Assertions.settingsGradle;
+
+class EnableDevelocityBuildCacheTest implements RewriteTest {
+
+    @Test
+    @DocumentExample
+    void addBuildCacheRemoteConfig() {
+        rewriteRun(spec -> spec.recipe(new EnableDevelocityBuildCache("true", "true", "System.getenv(\"CI\") != null")),
+          settingsGradle(
+            """
+              plugins {
+                  id 'com.gradle.develocity' version '3.17.6'
+              }
+              develocity {
+                  server = 'https://dev.example.com/'
+              }
+              """,
+            """
+              plugins {
+                  id 'com.gradle.develocity' version '3.17.6'
+              }
+              develocity {
+                  server = 'https://dev.example.com/'
+                  buildCache {
+                      remote(develocity.buildCache) {
+                          enabled = true
+                          push = System.getenv("CI") != null
+                      }
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void addBuildCacheLocalConfig() {
```

Comment:
The configuration example here isn't valid. Local doesn't accept the `develocity.buildCache` argument. Local is also default enabled with the caveat that the Gradle property or CLI switch must also be enabled for the feature to turn on.

Reply:
Fixed